docs/src/content/docs/guides/pages.mdx

```diff
@@ -106,13 +106,35 @@ The following properties differ from Markdown frontmatter:
 - The [`editUrl`](/reference/frontmatter/#editurl) option requires a URL to display an edit link.
 - The [`sidebar`](/reference/frontmatter/#sidebar) property is not supported. In Markdown frontmatter, this option allows customization of [autogenerated link groups](/reference/configuration/#sidebar), which is not applicable to pages using the `<StarlightPage />` component.
 
-{/* ##### `sidebar` */}
-
-{/* **type:** `SidebarEntry[] | undefined` */}
-{/* **default:** the sidebar generated based on the [global `sidebar` config](/reference/configuration/#sidebar) */}
-
-{/* Provide a custom site navigation sidebar for this page. */}
-{/* If not set, the page will use the default global sidebar. */}
+##### `sidebar`
+
+**type:** `SidebarEntry[]`  
+**default:** the sidebar generated based on the [global `sidebar` config](/reference/configuration/#sidebar)
+
+Provide a custom site navigation sidebar for this page.
+If not set, the page will use the default global sidebar.
+
+For example, the following page overrides the default sidebar with a link to the homepage and a group of links to different constellations.
+The current page in the sidebar is set using the `isCurrent` property and an optional `badge` has been added to a link item.
```

Comment:
The example is really explicit! Nicely done!

I am left wondering whether I can still do things like autogenerate here, though. If so, it might be nice to show another grouping where that happens? If not, a line mentioning that this config should manually list out the entire contents of the sidebar as autogeneration of groups is not available, or something like that, would be helpful!